### DISCUSSION TITLE
Two divs with same height, aligned on the smaller one with scroll bar

### DISCUSSION THREAD

**Discussion by laurent512** [tailwindlabs/tailwindcss#9945] [Help] [answered]
Hello, 

Looking for some help here, I am not css expert, please apologize for the potential easy question ^^'
I have a layout with two divs ; 

```
<div className="grid  grid-cols-3  ">
       <div className="col-span-2">
           ...CONTENT A
       </div>
       <div className="col-span">
           ...CONTENT B 
        </div>
</div>
```
The content B is longer than Content A. 
My objective is to have both div the same height adjusted on content A size, and add a scrollbar to scroll within content B frame. 

Exactly as shown here:  https://drive.google.com/file/d/1lBIZls661AuC3OSh36_JOywbKxwegVu5/view?usp=share_link

Could you provide me some guidance to achieve that ? thank you :)

**Comment by wongjn** [comment, marked as answer]
Try having the content B inside a vertical flex container. Have the content B wrapped by a single flex item with `0` `flex-basis` so that it does not affect the outer grid height. Apply `flex-grow: 1` to let it expand to the available space. [Example Tailwind Play](https://play.tailwindcss.com/y4WPevtVlI)

**Reply by laurent512** [reply]
Incredible, thank you so much !!!!

**Reply by armeltalla** [reply]
I was having the same issue, and this worked!! Thank you!!!

**Reply by sloan58** [reply]
Amazing - thank you!
